chimba/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "chimba",
+    "assets": [
+      {
+        "description": "The first blokchain in colombia",
+        "denom_units": [
+          {
+            "denom": "ucmba",
+            "exponent": 0
+          },
+          {
+            "denom": "cmba",
+            "exponent": 6
+          }
+        ],
+        "base": "ucmba",
+        "name": "Chimba",
+        "display": "cmba",
+        "symbol": "CMBA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/chimba-blue.png",
+          "svg":"https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/chimba-blue.svg"
+        },
+        "coingecko_id": "chimba-token"  
```

Comment:
please remove this CoinGecko id until it's been registered with CoinGecko. I could not find it in their list (https://api.coingecko.com/api/v3/coins/list)